regexes.go

```diff
@@ -60,4 +61,5 @@ var (
 	latitudeRegex            = regexp.MustCompile(latitudeRegexString)
 	longitudeRegex           = regexp.MustCompile(longitudeRegexString)
 	sSNRegex                 = regexp.MustCompile(sSNRegexString)
+	hostnameRegex            = regexp.MustCompile(hostnameRegexString)
```

Comment:
Also minor but if you could make the groups non-capturing that'd be great! eg. `(?:`